onnx/large_helper.py

```diff
@@ -0,0 +1,346 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import enum
+import os
+from typing import Any, Dict, Iterable, Optional, Set, Tuple
+
+import numpy as np
+
+from onnx import (
+    GraphProto,
+    ModelProto,
+    StringStringEntryProto,
+    TensorProto,
+    checker,
+    load_model,
+)
+from onnx.external_data_helper import (
+    ExternalDataInfo,
+    _get_all_tensors,
+    _sanitize_path,
+    uses_external_data,
+)
+from onnx.helper import make_model
+
+
+def make_large_tensor_proto(
+    location: str, name: str, tensor_type: int, shape: Tuple[int, ...]
+) -> TensorProto:
+    """
+    Create an external tensor.
+
+    :param location: unique identifier (not necessary a path)
+    :param name: initializer name in the graph
```

Comment:
nit: it should be "tensor name" instead of "initializer name" ... I assume we can use this everywhere a TensorProto is used, eg., as an attribute in Constant node.

Reply:
done